source/includes/v3/_products.md

```diff
@@ -391,6 +440,13 @@ woocommerce.post("products", data).parsed_response
       "T-shirts"
     ],
     "tags": [],
+    "brands": [
+      {
+        "id": 99,
+        "name": "test",
```

Comment:
Not a big deal, but I see in the `attributes`, the `name` property is capitalized:

<img width="308" height="215" alt="imatge" src="https://github.com/user-attachments/assets/72aff690-9290-4b8b-95f4-c6e17645ec66" />

I think it's nice to make it clear it's a formatted text instead of a slug in snake-case.